.evergreen/config.yml

```diff
@@ -45,11 +45,16 @@ functions:
         binary: bash
         args: [.evergreen/fetch-repo.sh]
 
+  "assume role":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${assume_role_arn}
+
   "execute tests":
     - command: subprocess.exec
       type: test
       params:
-        include_expansions_in_env: [DIR]
+        include_expansions_in_env: [DIR, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN]
```

Comment:
Is this simply for the introduction of blob storage? This has nothing to do with Bedrock, right?